example/examples/Alerts.tsx

```diff
@@ -65,6 +65,7 @@ export default class Alerts extends Component<Props, State> {
         <WebView
           source={{html: HTML}}
           automaticallyAdjustContentInsets={false}
+          useWebView2
```

Comment:
Since this example is for all platforms, I'm not sure we'll want to leave the useWebView2 prop in.

Reply:
I don't think this affects it since people building apps for Android, iOS and Windows need to include the prop to get WV2, it essentially just gets ignored by the Android and iOS version of the web view component, just like custom iOS and Android props get ignored by windows, I can go ahead and create a Alerts.windows.tsx but I'd say its not required.

Reply:
I added several windows specific components for the scenarios in question